src/templates/can-observable-object/observable-object-static-properties-output.js

```diff
@@ -0,0 +1,29 @@
+class Foo extends ObservableObject {
```

Comment:
Sorry, I didn't realize this earlier, but won't changing this to `class Foo` instead of `class Todo` break things that are using this variable? If this change isn't caused by this PR, we can fix it in a separate PR.